UPGRADING.md

```diff
@@ -3,4 +3,24 @@ Upgrading to Graylog 5.1.x
 
 ## New Functionality
 
+## Index Default Configuration
+Support for configuring system index initialization defaults, and ongoing in-database index set defaults has been added.
+Please see the corresponding change log entry [pr-13018.toml](https://github.com/Graylog2/graylog2-server/blob/master/changelog/5.1.0/pr-13018.toml)
```

Comment:
Let's not link to a changelog entry that doesn't exist yet. :slightly_smiling_face: We don't know yet if the first version it will appear in is 5.1.

I appreciate the detailed changelog details, but changelog entries shouldn't be too exhaustive or be a replacement for upgrade notes or documentation. Let's put that information into the docs or this UPGRADING document. Also, we currently don't render the user details anywhere.

Reply:
Ahh yes, makes sense. Thanks for the guidance on that @bernd. I'll move all of that bulk detail over to `UPGRADING.md`, and coordinate any relevant changes in the docs.

Thanks also for the reminder that the user details are not rendered anywhere right now. I'll keep that in mind. Is there a plan to render those at some point (like to a linkable details page)?

Reply:
> Thanks also for the reminder that the user details are not rendered anywhere right now. I'll keep that in mind. Is there a plan to render those at some point (like to a linkable details page)?

Not sure yet. It all depends on how we are going to render the changelogs in the future.

Reply:
> Let's put that information into the docs or this UPGRADING document.

@bernd This has been done. I have paired down the change log to just the `message`, and moved everything else to the `UPGRADING` doc.

Reply:
Thank you!